### ISSUE TITLE
RankExp

### ISSUE THREAD

**Issue by SteveKitten** [Bimmr/McInfected#158]
We are setting up voting and when voting we would like players to receive 100 exp in infection. The command says there has been an internal error. I had talked to you earlier and you said that this was no longer a problem in the newest version, but it continued to be one.

## What's Currently Happening
When doing /mc admin RankExp +100 it says "An internal error occurred while attempting to perform this command."

----
## Error Code
```
01.08 16:55:26 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at me.bimmr.bimmcore.commands.CommandHandler.onCommand(CommandHandler.java:68) ~[?:?]
01.08 16:55:26 [Server] INFO at me.bimmr.bimmcore.commands.SuperCommand.execute(SuperCommand.java:27) ~[?:?]
01.08 16:55:26 [Server] INFO at com.bimmr.mcinfectedranks.Commands.AdminRankExpCommand.execute(AdminRankExpCommand.java:29) ~[?:?]
01.08 16:55:26 [Server] INFO Caused by: java.lang.ArrayIndexOutOfBoundsException: 3
01.08 16:55:26 [Server] INFO at java.lang.Thread.run(Unknown Source) [?:1.8.0_77]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:557) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:654) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:374) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:715) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_77]
01.08 16:55:26 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_77]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:1) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.PacketPlayInChat.a(PacketPlayInChat.java:45) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:997) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at net.minecraft.server.v1_8_R3.PlayerConnection.handleCommand(PlayerConnection.java:1162) [spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at org.bukkit.craftbukkit.v1_8_R3.CraftServer.dispatchCommand(CraftServer.java:641) ~[spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot_1.8.8_latest.jar:git-Spigot-21fe707-e1ebe52]
01.08 16:55:26 [Server] INFO org.bukkit.command.CommandException: Unhandled exception executing command 'mc' in plugin McInfected v1.5.1
01.08 16:55:26 [Server] ERROR null`

```

----
## Your Environment

* Spigot Version used: 1.8.8
* McInfected Version: 1.5.1
* BimmCore Version: 0.0.8
* Any Addons?(Include their versions)
Ranks: V1.2.3
Lobby Items: V1.0.0
Leaderboard: V0.1.1
Custom Signs: V0.0.6

**Comment by Bimmr** [comment]
I'll look into it this weekend, I have an idea of what's causing it.

**Comment by SteveKitten** [comment]
Thank you very much, it's really the only thing preventing our server from opening completely.

**Comment by Bimmr** [comment]
Fixed in McInfected Ranks - [v1.2.4]
